tests/bud.bats

```diff
@@ -366,6 +366,24 @@ _EOF
   expect_output --substring "invalid response status"
 }
 
+@test "build test has gid in supplemental groups" {
+  _prefetch alpine
+  run_buildah build $WITH_POLICY_JSON -t source -f $BUDFILES/supplemental-groups/Dockerfile
+  # gid 1000 must be in supplemental groups
+  expect_output --substring "Groups:	1000"
+}
+
+@test "build test if supplemental groups has gid with --isolation chroot" {
+  skip_if_rootless_environment
+  skip_if_no_runtime
```

Comment:
Are either of these conditions required for this test?

Reply:
Other tests which have `--isolation chroot` are being skipped for similar scenarios that's why i added it, I can try removing it.

Reply:
I think we should probably try to remove them from all of the chroot tests.

Reply:
Removed from this test, lets see if CI is happy. I'll do this for all the other tests in a different PR since they are not part of this PR.